rbac/management/principal/proxy.py

```diff
@@ -113,8 +113,14 @@ def _get_proxy_service(self):  # pylint: disable=no-self-use
         }
         return proxy_conn_info
 
-    def _request_principals(self, url, account=None, account_filter=False, method=requests.get, params=None, data=None):
+    def _request_principals(self, url, account=None, account_filter=False, method=requests.get,  # noqa: C901
+                            params=None, data=None):
         """Send request to proxy service."""
+        if settings.BYPASS_BOP_VERIFICATION:
+            to_return = []
+            for principal in data['users']:
```

Comment:
@j00bar - Testing this locally, and noticed one minor thing. If we call `_request_principals()` from `request_filtered_principals()` this works as expected, because `payload['users']` will always exist.

However, for the `/api/rbac/v1/principals/` endpoint, we call this from `request_principals()`, which does not filter the users, [so will never have `payload['users']` populated](https://github.com/RedHatInsights/insights-rbac/pull/265/files#diff-42895888fc7b254a2fa368acc404c6feR204). 

Maybe we just need a check to ensure `data['users']` exists before appending the principals? Also, for `/api/rbac/v1/principals/`, [the view is expecting a `status_code` on the response object](https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/management/principal/view.py#L156), so we may want to just add `status=200` to that dictionary we return?